lib/elixir/pages/references/naming-conventions.md

```diff
@@ -38,7 +38,7 @@ Function names may also start with an underscore. Such functions are never impor
     iex> _wont_be_imported()
     ** (CompileError) iex:1: undefined function _wont_be_imported/0
 
-Due to this property, Elixir relies on functions starting with underscore to attach compile-time metadata to modules. Such functions are most often in the `__foo__` format. For example, every module in Elixir has an [`__info__/1`](`c:Module.__info__/1`) function:
+Due to this property, Elixir relies on functions starting with underscore to attach compile-time metadata to modules. Such functions are most often in the `__foo__` format. For example, every module in Elixir has an [`__info__/1`](https://hexdocs.pm/elixir/main/Module.html#c:__info__/1) function:
```

Comment:
do you have a live example where this link was broken? It works here:

<img width="1154" alt="image" src="https://github.com/elixir-lang/elixir/assets/76071/bd9a4a1d-e3ac-4f9d-b276-a8eb2d5bf776">

(https://hexdocs.pm/elixir/main/naming-conventions.html)

There's quite a bit of logic in ExDoc to make these links work but there're sometimes bugs and we'd love to learn about them so we can fix them.

Reply:
Oh, is it that you're looking at

https://github.com/elixir-lang/elixir/blob/main/lib/elixir/pages/references/naming-conventions.md

and _not_:

https://hexdocs.pm/elixir/main/naming-conventions.html

then yeah, that's a problem.

Reply:
If we'd move forward with this, we should drop main from the url, i.e.:

```diff
- https://hexdocs.pm/elixir/main/naming-conventions.html
+ https://hexdocs.pm/elixir/naming-conventions.html
```